.travis.yml

```diff
@@ -11,30 +11,16 @@ cache:
 
 env:
   global:
-    - COMPOSER_ARGS="--no-interaction"
+    - COMPOSER_ARGS="--no-interaction --ignore-platform-reqs"
     - COVERAGE_DEPS="php-coveralls/php-coveralls"
 
 matrix:
   fast_finish: true
   include:
-    - php: 7.1
-      env:
-        - DEPS=lowest
-    - php: 7.1
-      env:
-        - DEPS=latest
-        - CS_CHECK=true
-        - TEST_COVERAGE=true
-    - php: 7.2
-      env:
-        - DEPS=lowest
-    - php: 7.2
-      env:
-        - DEPS=latest
     - php: 7.3
       env:
         - DEPS=lowest
-    - php: 7.3
+    - php: 8.0
```

Comment:
Actually this file should be deleted too, but it is still in instructions.

Why did you replace PHP 7.4 instead of adding 8.0?

Reply:
Sorry for this one...
I checked other component's Pull-Request, for example: [laminas-di](https://github.com/laminas/laminas-di/pull/15/commits)

```
Actually this file should be deleted too, but it is still in instructions.
```
I will delete this file.